src/main/java/com/moodybluez/enterprise/dao/IEntryDAO.java

```diff
@@ -5,6 +5,13 @@
 import java.util.List;
 import java.util.Map;
 
+/**
+ * Allows the user to enter their entry for the day
+ *
+ * @param year the year which the entry was entered
+ * @param month the month which the entry was entered
+ *
+ */
```

Comment:
for interfaces comment on the specific methods within the interface for example

/**
     * Creates a new Entry database record from a Entry object.
     *
     * Returns null if an error occurs or returns with entry if entry is saved in database
     *
     * @param entry Entry object representing a entry to be created
     * @return newly created Entry object
     */
this would go on line 16